### PR TITLE
Add more Cosmos configs

### PR DESCRIPTION
**Pull request by Universal-Omega**
<s>Due to the amount of useful configs for the Cosmos skin, I did move it to a new "Cosmos" tab in ManageWikiSettings. Whether this is allowed or not, I am unsure, so if not, I will revert this back to the styling tab.  I will submit MirahezeMagic PR  to add `⧼settings-cosmos⧽` if this is approved.</s> reverted

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -186,12 +186,36 @@
...
+	'wgCosmosContentBackgroundColor' => [
+		'default' => '#000',
+	],
+	'wgCosmosBannerBackgroundColor' => [
+		'default' => '#001e3b',
+	],
+	'wgCosmosLinkColor' => [
+		'default' => '#006cb0',
+	],
+	'wgCosmosBackgroundImageNorepeat' => [
```
Comment (Reception123) [inline LocalSettings.php:210]: overridedefault is true but here the default is false

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply LocalSettings.php:210]: That's intential. Paladox told me that that's how the boolean values have to work, as he changed all the other ones to do the same.

**Review by Reception123 — APPROVED** [review]
Per Paladox's previous merge

**Comment by JohnFLewis** [comment]
That is wrong, override default MUST match LocalSettings.

**Comment by Universal-Omega** [comment]
When they were the opposite of what they are now, the configs did not work and we're force setting themselves. Cosmos coding is a bit different requiring a small change to ManageWikiPermissions possibly I guesz possibly.

**Pull request merged**